src/components/EditorWindow.tsx

```diff
@@ -127,7 +131,20 @@ const EditorWindow: FunctionComponent<EditorWindowProps> = ({
         // As currentServer isn't necessarily set when we bind this and this ends up out of scope we
         // can make the rest happen in React land by just triggering the click action on the play button.
         // Yes it's a hack, and I don't like it, but it works
-        if (goRef && goRef.current) goRef.current.click();
+        if (goRef && goRef.current) {
+          goRef.current.dataset.queryMode = "currentLine";
+          goRef.current.click();
+        }
+      }
+    );
+
+    editor.addCommand(
+      monaco.KeyMod.CtrlCmd | monaco.KeyCode.KEY_E,
+      function () {
+        if (goRef && goRef.current) {
+          goRef.current.dataset.queryMode = "selection";
```

Comment:
Should this be `QueryMode.selection`?

Reply:
Hmm, possibly yes, I guess as it's a string enum it should translate properly.